aerosonicdb/features/extract_mfccs.py

```diff
@@ -1,18 +1,23 @@
-import os
```

Comment:
just some formatting and switching to use `click`, added `--force` optional flag..